fenicsxprecice/adapter_core.py

```diff
@@ -85,49 +86,48 @@ def determine_function_type(input_obj):
         raise Exception("Error determining type of given dolfin FunctionSpace")
 
 
-def convert_fenicsx_to_precice(fenicsx_function, local_ids):
+def convert_fenicsx_to_precice_coordinateBased(fenicsx_function, local_coords):
```

Comment:
```suggestion
def convert_fenicsx_to_precice_coordinate_based(fenicsx_function, local_coords):
```

Is this renaming really necessary? If the new name is more appropriate we can do the rename.

Note that the main reason why I do not want to rename is that renaming also requires an update for the unit tests.

Reply:
No, the renaming isn't necessary. I will change that.